server/src/main.rs

```diff
@@ -45,6 +50,25 @@ pub fn run_openapi() -> Result<(), String> {
         .map_err(|e| e.to_string())
 }
 
+fn setup_vnc(log: &Logger) -> VncServer<PropolisVncServer> {
+    // XXX: Do we want to specify this information in the config file?
+    let config = VncServerConfig {
+        addr: SocketAddr::new(IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)), 5900),
+        version: ProtoVersion::Rfb38,
+        // XXX: vncviewer won't work without offering VncAuth, even though it doesn't ask to use
+        // it.
```

Comment:
Can probably be a "durable" (sans XXX) comment, considering the value

Reply:
Fair. I don't think I care too much about vncviewer as a client (it seems to be buggy, or at least behaves differently from others when processing framebuffer updates), but it has the nice quality of very verbose error messages, which has helped me debug protocol errors on my end. VNCAuth isn't implemented yet (and maybe never will be) so it probably should be removed in that case.

Reply:
Addressed in ddc78a6